### ISSUE TITLE
docker-cloud stack create results in get_environ_proxies() takes exactly 2 arguments (1 given)

### ISSUE THREAD

**Issue by l0rn** [docker/dockercloud-cli#52]
Hello, in v1.0.7 of the docker cloud cli the command below stopped working
```
$ docker-cloud stack create -f ci/test_stackfile.yml -n teststack     
get_environ_proxies() takes exactly 2 arguments (1 given)
$ docker-cloud -v    
docker-cloud 1.0.7
```

The error message is somewhat unverbose

**Comment by l0rn** [comment]
I figured it out: problem is requests. 2.14.0 made a breaking change. docker cloud should fix it or freeze the dependency to 2.13.0

**Comment by freekrai** [comment]
We encountered this message too and it broke all deployments from Travis to Docker Cloud.

**Comment by tifayuki** [comment]
I will do a patch to this ASAP

**Comment by tifayuki** [comment]
Fixed in the new version v1.0.8
